### PR TITLE
[MIBM-478] DeclarationNotFound page

### PR DESCRIPTION
**Pull request by andy1138**
Back and try again links need fixing

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -61,6 +61,17 @@ invalidRequest.li2 = Make a declaration if you are:
 invalidRequest.Import.restart = bringing commercial goods into Great Britain
 invalidRequest.Export.restart = taking commercial goods out of Great Britain
 
+# DeclarationNotFound
+declarationNotFound.title = Are you bringing in goods from EU countries, non-EU countries or both?
+declarationNotFound.heading = Declaration not found
```
Comment (sureshhmrc) [inline conf/messages.en:66]: title and heading should generally be the same, please double check with Ben

File: app/uk/gov/hmrc/merchandiseinbaggage/views/DeclarationNotFoundView.scala.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+@layout(pageTitle = Some(messages("declarationNotFound.title")), maybeBackButtonUrl = Some(ExciseAndRestrictedGoodsController.onSubmit), maybeDeclarationType = None) {
+    @h1(("declarationNotFound.heading"))
+    @p(Html(messages("declarationNotFound.li1")))
+    @p(Html(messages("declarationNotFound.li2")))
+    @p(Html(messages("declarationNotFound.li3")))
+    <p class="govuk-body">
+        @messages("declarationNotFound.li4.a.text")
+        @link("TODO /retrieve-declaration", "declarationNotFound.li4.b.text")
+        @messages("declarationNotFound.li4.c.text")
+        @link( ExciseAndRestrictedGoodsController.onSubmit.url, "declarationNotFound.li4.d.text")
```
Comment (sureshhmrc) [inline app/uk/gov/hmrc/merchandiseinbaggage/views/DeclarationNotFoundView.scala.html:43]: if its for a new journey, this link should point to /goods-destination or something

File: conf/messages.en (hunk 1 above)
Comment (andy1138) [reply conf/messages.en:66]: Both bits of text are cut/paste from the prototype, but can double check with Ben

Comment (jordanrowe) [reply conf/messages.en:66]: The prototype titles are never right, I have raised it previously but it's fallen on deaf ears

File: app/uk/gov/hmrc/merchandiseinbaggage/views/DeclarationNotFoundView.scala.html (hunk 2 above)
Comment (andy1138) [reply app/uk/gov/hmrc/merchandiseinbaggage/views/DeclarationNotFoundView.scala.html:43]: This is what the ticket and prototype require /excise-and-restricted-goods

File: conf/messages.en (hunk 1 above)
Comment (andy1138) [reply conf/messages.en:66]: @jordanrowe do you still want me to check with Ben? or just make them match

File: app/uk/gov/hmrc/merchandiseinbaggage/views/DeclarationNotFoundView.scala.html (hunk 2 above)
Comment (jordanrowe) [reply app/uk/gov/hmrc/merchandiseinbaggage/views/DeclarationNotFoundView.scala.html:43]: I think Suresh is right, first page in the journey is /goods-destination -> our prototype is becoming a mess

regardless of that, this link is going to the POST endpoint, you need to go to the .onPageLoad() method

File: conf/messages.en (hunk 1 above)
Comment (jordanrowe) [reply conf/messages.en:66]: Make them match, but I'll bring it up in standup today. Will be more vigilant in refinements as well, when we are going through the prototype I'll try and stay awake and call out these types of things

File: app/uk/gov/hmrc/merchandiseinbaggage/views/DeclarationNotFoundView.scala.html (hunk 2 above)
Comment (andy1138) [reply app/uk/gov/hmrc/merchandiseinbaggage/views/DeclarationNotFoundView.scala.html]: change to /goods-destination

**Review by sureshhmrc — APPROVED** [review]

**Pull request merged**